packages/manager/apps/web/client/app/components/manager-order-payment-means/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+import component from './manager-order-payment-means.component';
+
+const moduleName = 'ovhManagerOrderPaymentMeans';
+
+angular
+  .module(moduleName, [])
+  .component('ovhManagerOrderPaymentMeans', component)
```

Comment:
Why not update https://github.com/ovh/manager/tree/master/packages/components/ng-ovh-payment-method if it doesn't suit all the needs ?

Reply:
Maybe this component should be part of `@ovh-ux/ng-ovh-payment-method` ?

Reply:
Here it is : https://github.com/ovh/manager/pull/1871